changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 3.4.4
+----------
+- [PATCH] Disregard pageload errors for the non-primary frame during interactive auth (#1357)
```

Comment:
Based on the android documentation all errors (images, scripts, iframes) that fail to load cause a page load error.